src/Sulu/Bundle/AudienceTargetingBundle/Tests/Functional/Controller/TargetGroupControllerTest.php

```diff
@@ -374,15 +379,17 @@ public function testSingleDelete()
 
         $this->getEntityManager()->flush();
 
-        $client = $this->createAuthenticatedClient();
-        $client->request('DELETE', self::BASE_URL . '/' . $targetGroup->getId());
+        $targetGroupId = $targetGroup->getId();
```

Comment:
Is there any specific reason to extract that into a separate variable? In tests I usually try to avoid something like this, because if the `id` of the target group changes for some reason the testing logic might be flawed.

Reply:
commented below. basically `$targetGroup->getId();` will return null after it was removed because they share now the same doctrine unit of work.